content/guides/core/automations/automation-events.md

```diff
@@ -86,6 +84,14 @@ For **Run metrics change threshold met** events, you can configure:
 
 The current and prior window do not overlap. By default, the current window includes the 10 most recent logged values, and the prior window includes the 50 logged values prior to the current window.
 
+#### Run filters
+This section describes how the automation selects runs to evaluate.
+
+- Each run is considered separately, and the automation can potentially start due to the result of multiple runs.
```

Comment:
I find this sentence a bit confusing and a bit repetitive with lines 91 and 92 - I wonder if we can just drop this line and move line 92 first, maybe tweaking it to say something like "if you specify a run filter, all matching run will be evaluated by the automation. If no run filter is specified all runs in a project will be evaluated."

Reply:
PTAL

Reply:
Nope, I would have missed this. Addressed.